prow/flagutil/github.go

```diff
@@ -49,6 +62,7 @@ func (o *GitHubOptions) AddFlagsWithoutDefaultGitHubTokenPath(fs *flag.FlagSet)
 }
 
 func (o *GitHubOptions) addFlags(wantDefaultGitHubTokenPath bool, fs *flag.FlagSet) {
+	fs.StringVar(&o.host, "github-host", github.DefaultHost, "GitHub's default host (may differ for enterprise)")
```

Comment:
This is specifically a `git` host, right?

Reply:
No, or not only as a git host, that's the host for deriving github oauth endpoint, too.